tests/foreman/api/test_multiple_paths.py

```diff
@@ -214,9 +214,8 @@ def test_positive_post_status_code(self):
                 # partials cannot be compared for class identity and the class
                 # hierarchy needs fixing (SatelliteQE/nailgun#42), so we just
                 # comment it out above.
-                if (entity_cls in BZ_1118015_ENTITIES and
-                        bz_bug_is_open(1118015)):
-                    self.skipTest('Bugzilla bug 1118015 is open.')
+                if entity_cls in BZ_1118015_ENTITIES and is_open('BZ:1118015'):
+                    continue  # pytest can't skip inside a subTest.
```

Comment:
As I understood we are removing the skipping part. because of "# pytest can't skip inside a subTest." ? can we please some logger statement as well.